webbpsf/otelm/SMIF_hexike.csv

```diff
@@ -0,0 +1,16 @@
+Type,SM Mode,H3 (45-astig),H4 (Focus),H5 (0-astig),H6 (X coma),H7 (Y coma),H8 (3rd Sphr)

```

Comment:
Ideally we should add a header line in these files with a `# ` to say what they are.

Reply:
Done.